src/js/containers/aboutTheData/AgenciesContainer.jsx

```diff
@@ -122,8 +123,9 @@ const AgenciesContainer = ({
         }
         const newPage = goToFirstPage ? 1 : publicationsPage;
         setLoading([false, false, true]);
-        // Get the (cumulative) total budgetary resources from the latest period available
-        const federalTotal = getFederalBudget(federalTotals, selectedFy);
+        // Get the (cumulative) total budgetary resources from the latest (revealed) period
+        const latestPeriod = getLatestPeriod(submissionPeriods.toJS(), selectedFy).period;
+        const federalTotal = getFederalBudget(federalTotals, selectedFy, latestPeriod);
```

Comment:
latest period includes the year in it IIRC so perhaps this fn only needs `federalTotals, latestPeriod`?